src/views/layouts/application.slang

```diff
@@ -8,6 +8,7 @@ html
 
     link rel="stylesheet" href="/stylesheets/main.css"
     link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css"
```

Comment:
Would be cool to use some dark theme. Maybe `ocean.min.css` ?
You can find it here https://highlightjs.org/static/demo/